### PR TITLE
Improvements to the entire codebase

### PR DESCRIPTION
**Pull request by Atmos4**
- rephrase all code and files from french to english
- design improvements
- move forward with events, adding basic form structure for changing and submitting an entry

### CODE REVIEW

File: pages/event_submit.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                <label for="entryComments">
+                    Remarques:
+                </label>
+                <textarea name="event_comments"></textarea>
+            </fieldset>
+            <h4>Courses : </h4>
+            <table role="grid">
+                <?php foreach ($competitions as $competition) : ?>
+                    <tr class="display">
+                        <td class="competition-entry"><i class="fas fa-arrow-right"></i></td>
```
Comment (Mathieu414) [inline pages/event_submit.php:65]: Not sure if the arrow is useful here. Without anything could be good enough in my opinion.

**Review by Mathieu414 — APPROVED** [review]
Appart from the arrow thing, LGTM good job my man

File: pages/event_submit.php (hunk 1 above)
Comment (Atmos4) [reply pages/event_submit.php]: Removed!

**Review by Mathieu414 — APPROVED** [review]
LGTM

**Pull request merged**